include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -19,6 +19,13 @@
 
 #include <CL/sycl.hpp>
 
+// Macros to check the new SYCL features
+#if defined(__LIBSYCL_MAJOR_VERSION) && defined(__LIBSYCL_MINOR_VERSION)
+#    define _ONEDPL_KERNEL_BUNDLE_PRESENT (__LIBSYCL_MAJOR_VERSION >= 5 && __LIBSYCL_MINOR_VERSION >= 2)
```

Comment:
The only question I have is where do you find actual values for macros? Is it documented anywhere?
I'm asking because I'm wondering will it work for `dpcpp` and `clang` with `-fsycl` from https://github.com/intel/llvm both

Reply:
I checked the values with one of last compiler which has already produced "the warnings"..
The macros are present in the opensource compiler version as well.

Reply:
Thanks. No more questions.
Version was changed here (https://github.com/intel/llvm/pull/3538)
Corresponding changes for Compiler were made here (https://github.com/intel/llvm/pull/3464)
Difference is 1 week. I think it's not critical to check build date